.gitignore

```diff
@@ -14,6 +14,7 @@ docs/build/
 .cache
 .runtest_log.json
 dmypy.json
+.pytest_cache/
```

Comment:
You can add this to you own global .gitignore -- I don't see a reason to include this in mypy's .gitignore.